From 45a9a57c252306d9416e807bd98f820d8449d960 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sat, 28 Mar 2020 13:33:02 +0200
Subject: [PATCH 3/4] shrink-shrunk v3

	modified:   lineage/res/res/values-fr/strings.xml
	modified:   lineage/res/res/values-ru/strings.xml
	modified:   lineage/res/res/values/strings.xml
---
 lineage/res/res/values-fr/strings.xml | 8 ++++----
 lineage/res/res/values-ru/strings.xml | 8 ++++----
 lineage/res/res/values/strings.xml    | 8 ++++----
 3 files changed, 12 insertions(+), 12 deletions(-)

diff --git a/lineage/res/res/values-fr/strings.xml b/lineage/res/res/values-fr/strings.xml
index 2436179e..04279b9b 100644
--- a/lineage/res/res/values-fr/strings.xml
+++ b/lineage/res/res/values-fr/strings.xml
@@ -97,10 +97,10 @@
     <string name="lineage_vendor_security_patch">Niveau du correctif de sécurité du fournisseur</string>
     <string name="unknown">Inconnu</string>
     <string name="app_killed_message">Application arrêtée</string>
-    <string name="kilobitspersecond_short">kb/s</string>
-    <string name="megabitspersecond_short">Mb/s</string>
-    <string name="kilobytespersecond_short">ko/s</string>
-    <string name="megabytespersecond_short">Mo/s</string>
+    <string name="kilobitspersecond_short">k</string>
+    <string name="megabitspersecond_short">m</string>
+    <string name="kilobytespersecond_short">K</string>
+    <string name="megabytespersecond_short">M</string>
     <string name="trust_feature_name">Trust</string>
     <string name="permlab_trustInterface">accéder à l\'interface Trust</string>
     <string name="permdesc_trustInterface">Autorise une application à afficher des avertissements et suggestions Trust</string>
diff --git a/lineage/res/res/values-ru/strings.xml b/lineage/res/res/values-ru/strings.xml
index 816f21c8..de58c133 100644
--- a/lineage/res/res/values-ru/strings.xml
+++ b/lineage/res/res/values-ru/strings.xml
@@ -97,10 +97,10 @@
     <string name="lineage_vendor_security_patch">Состояние исправлений безопасности от производителя</string>
     <string name="unknown">Неизвестно</string>
     <string name="app_killed_message">Приложение завершено</string>
-    <string name="kilobitspersecond_short">кбит/с</string>
-    <string name="megabitspersecond_short">Мбит/с</string>
-    <string name="kilobytespersecond_short">кБ/с</string>
-    <string name="megabytespersecond_short">МБ/с</string>
+    <string name="kilobitspersecond_short">к</string>
+    <string name="megabitspersecond_short">м</string>
+    <string name="kilobytespersecond_short">К</string>
+    <string name="megabytespersecond_short">М</string>
     <string name="trust_feature_name">Trust</string>
     <string name="permlab_trustInterface">доступ к интерфейсу Trust</string>
     <string name="permdesc_trustInterface">Позволяет приложению отображать предупреждения и предложения Trust</string>
diff --git a/lineage/res/res/values/strings.xml b/lineage/res/res/values/strings.xml
index c632f426..0d603cd8 100644
--- a/lineage/res/res/values/strings.xml
+++ b/lineage/res/res/values/strings.xml
@@ -152,10 +152,10 @@
     <string name="app_killed_message">Application killed</string>
 
     <!-- Status bar network traffic monitor strings -->
-    <string name="kilobitspersecond_short">kb/s</string>
-    <string name="megabitspersecond_short">Mb/s</string>
-    <string name="kilobytespersecond_short">kB/s</string>
-    <string name="megabytespersecond_short">MB/s</string>
+    <string name="kilobitspersecond_short">k</string>
+    <string name="megabitspersecond_short">m</string>
+    <string name="kilobytespersecond_short">K</string>
+    <string name="megabytespersecond_short">M</string>
 
     <!-- Trust interface -->
     <!-- This string will be referenced from other apps when they're referring to the Trust interface.
-- 
2.30.2

